### PR TITLE
Patch multiple occurrence of templates in a single string.

### PR DESCRIPTION
**Pull request by Rot127**

### CODE REVIEW

File: llvm/utils/TableGen/PrinterCapstone.cpp [hunk 1]
```diff
@@ -662,7 +662,7 @@ std::string edgeCaseTemplArg(std::string &Code) {
   PrintFatalNote("Edge case for C++ code not handled: " + Code);
 }
 
-static std::string handleDefaultArg(const std::string &TargetName,
+std::string handleDefaultArg(const std::string &TargetName,
```
Comment (XVilka) [inline llvm/utils/TableGen/PrinterCapstone.cpp:665]: Why not `static` anymore?

File: llvm/utils/TableGen/PrinterCapstone.cpp [hunk 2]
```diff
@@ -674,48 +674,53 @@ static std::string handleDefaultArg(const std::string &TargetName,
...
-      return DecName + "_" + Func.second + Rest;
+      Code = DecName + "_" + Func.second + Rest;
     }
   }
   // No template function, false positive or fixed
   return Code;
 }
 
-static void patchTemplateArgs(const std::string &TargetName,
+void patchTemplateArgs(const std::string &TargetName,
```
Comment (XVilka) [inline llvm/utils/TableGen/PrinterCapstone.cpp:695]: Same

**Review by XVilka — APPROVED** [review]
The change is LGTM.

File: llvm/utils/TableGen/PrinterCapstone.cpp (hunk 1 above)
Comment (Rot127) [reply llvm/utils/TableGen/PrinterCapstone.cpp:665]: Its part of an anonymous namespace. Doesn't need it.

**Review by XVilka — APPROVED** [review]

**Pull request merged**